prediction_market_agent_tooling/logprobs_parser.py

```
@@ -1,29 +1,39 @@
+    token: str
+    logprob: float
+    prob: float
+
+
+class FieldLogprobs(BaseModel):
+    key: str
+    logprobs: list[LogprobDetail]
 
 
 class LogprobsParser:
+    def __init__(self, skip_fields: list[str] = []):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)